src/onDraw/addRowDisplayToggle.js

```diff
@@ -25,17 +29,15 @@ export default function addRowDisplayToggle() {
             .map(d);
     }
 
-    var childNest = iterateNest(chart.data.raw, 0);
+    var childNest = iterateNest(chart.data.summarized, 0);
```

Comment:
Generally speaking, throw a comment that describes each data structure, how it's nested, whether it's an object or an array, etc.  There's just so much going on with the expandable rows that I have to figure out what's happening at each step. /whining

Reply:
Makes perfect sense - ill add that in